nixos/modules/services/hardware/lirc.nix

```diff
@@ -19,12 +19,29 @@ in {
           [lircd]
           nodaemon = False
         '';
+        default = ''
+          [lircd]
+          nodaemon        = False
+          driver          = default
+          device          = /dev/lirc0
+          output          = /run/lirc/lircd
+          pidfile         = /run/lirc/lircd.pid
+          plugindir       = ${pkgs.lirc}/lib/lirc/plugins
+          permission      = 666
+          allow-simulate  = No
+          repeat-max      = 600
+
+          [lircmd]
+          uinput          = False
+          nodaemon        = False
```

Comment:
To bad this was not added in the first place. I would prefer having sane default options. that can be extended by `extraOptions`.

Reply:
I think for now this is alright, can always add that later

Reply:
agreed.